README.md

```diff
@@ -1,18 +1,14 @@
 # Security Special Interest Group
 
-Covers horizontal security initiatives for the OpenTelemetry project, including
-regular security audits, the vulnerability management process, cross-cutting
-security documentation, and security community management.
+Covers horizontal security initiatives for the OpenTelemetry project, including regular security audits, the
+vulnerability management process, cross-cutting security documentation, and security community management.
```

Comment:
Sorry about the unrelated changes, I was going crazy with the break at the 80th character...